invite-to-slack/README.md

```diff
@@ -1,2 +1,11 @@
 # Invite Requests
 This directory contains the script we are using in order to receive invite requests through Slack itself. This make the invitation management much much easier for admins to manage 💃
+
+# Installation
+
+* Create a Slack invoming webhook for your Slack team: https://api.slack.com/incoming-webhooks
+* Open your Invitations Google Form, go to Settings and locate `Script editor` ![Script Editor](http://image.prntscr.com/image/a69f445f4d5143449ab30b3961eb219b.png)
```

Comment:
This is actually _not_ in the settings, but in the `More` section. 
You can phrase it like so:

Open your Invite request Google Form, and open up the `Script Editor`. In the current version of Google Forms it is located on the 'More' button like so:
![screen shot 2017-04-24 at 15 47 41](https://cloud.githubusercontent.com/assets/1665273/25342930/6fba7d28-2905-11e7-9058-3509e8bc262d.png)